xiblint/rules/named_colors.py

```diff
@@ -24,16 +28,26 @@ def check(self, context):  # type: (XibContext) -> None
             if element.parent.tag == 'namedColor':
                 continue
 
+            # Skip <color> tags part of a system color definition
+            if element.parent.tag == 'systemColor':
+                continue
+
             # Skip colors with alpha (if configured)
             if ignore_alpha and element.get('alpha') != '1':
                 continue
 
+            # If `systemColor` or `catalog` is present, it's a named system color
+            if element.get('systemColor') is not None or element.get('catalog') is not None:
+                if not allow_system_colors:
```

Comment:
Let's flip these checks for better performance
```suggestion
            if not allow_system_colors:
                if element.get('systemColor') is not None or element.get('catalog') is not None:
```

Reply:
Wait, forgot about the `continue`, don't change this